### PR TITLE
add studio tasks description 

### PR DESCRIPTION
**Pull request by MaelAudren**

### CODE REVIEW

File: src/docs/user/references/Tasks.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
+This section gathers the different scripts engine available and the operationg where they run.
```
Comment (marcocast) [inline src/docs/user/references/Tasks.adoc]: engines

File: src/docs/user/references/Tasks.adoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+|Ruby
+|Linux, Mac, Windows
+
+|Language R
+|Linux, Mac, Windows
+
+|===
+
+This section describes the different predefined tasks available and the operationg where they run.
```
Comment (marcocast) [inline src/docs/user/references/Tasks.adoc]: operationg ?

Comment (laurianed) [inline src/docs/user/references/Tasks.adoc]: available predefined tasks

File: src/docs/user/references/Tasks.adoc (hunk 1 above)
Comment (laurianed) [inline src/docs/user/references/Tasks.adoc]: available scripts engines

File: src/docs/user/references/Tasks.adoc [hunk 3]
```diff
@@ -0,0 +1,52 @@
+This section gathers the available scripts engines and the operationg system where they run.
```
Comment (marcocast) [inline src/docs/user/references/Tasks.adoc]: operationg

File: src/docs/user/references/Tasks.adoc [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+|Predefined Task Name
+|Description
+|Operating System
+
+|Cron
+|Task executed periodically using a cron syntax
+|Linux, Mac, Windows
+
+|LDAP Query
+|Provide an LDAP client for querying an LDAP server
```
Comment (marcocast) [inline src/docs/user/references/Tasks.adoc]: Provides

**Pull request merged**